cpu/cc26xx-cc13xx/rf-core/ieee-mode.c

```diff
@@ -1101,6 +1101,8 @@ off(void)
 
   while(transmitting());
 
+  /* stopping the rx explicitly results in lower sleep-mode power usage */
+  rx_off();
```

Comment:
I thought about this a bit more after posting. Perhaps `CMD_FS_POWERDOWN` can only be accepted if there are no running ops, I'll have a test myself when I get a chance to see what the command returns if we don't abort first.